docs/howto/operate/new-cluster/aws.md

```diff
@@ -184,9 +161,12 @@ have least amount of permissions possible.
    actually encrypted by `sops` before checking it in to the git repo. Otherwise
    this can be a serious security leak!
 
-2. Grant the freshly created IAM user access to the kubernetes cluster.
+2. Grant the freshly created IAM user access to the kubernetes cluster. As this requires
```

Comment:
We should renumber these. This should be 5? I guess it was already wrong :D

Reply:
All the numbering looks fine to me on the preview? https://2i2c-pilot-hubs--1768.org.readthedocs.build/en/1768/howto/operate/new-cluster/aws.html